internal/commands/fetch.go

```diff
@@ -200,12 +200,8 @@ func findMostRecentMatchingVersion(versions []model.FoundVersion, ver string) (i
 // sortFoundVersionsDesc sorts by semver then timestamp in descending order, ie. from newest to oldest
 func sortFoundVersionsDesc(versions []model.FoundVersion) {
```

Comment:
I did some tests and I think, actually it needs no additional sorting here as its already correctly sorted by
` NewVersionsCommand().ListVersions -> union.Versions ->  model.MergeFoundVersions`.
If not already exists, a test for fetch.go could cover this.

But if we want to keep an additional sorting in fetch.go (just to be safe), than we could extract the sorting 
function in `search.go -> MergeFoundVersions` and reuse it. 
Of course in fetch.go the comparison on tmId.name would be useless (will return always equal), but I think it is acceptable

Reply:
Agreed. I've removed the extra sorting. I believe the existing tests cover this well enough.